docs/yaml_configuration.rst

```diff
@@ -155,7 +155,10 @@ Kubernetes
 -  **namespace**: (optional) Kubernetes namespace where Patroni pod is running. Default value is `default`.
 -  **labels**: Labels in format ``{label1: value1, label2: value2}``. These labels will be used to find existing objects (Pods and either Endpoints or ConfigMaps) associated with the current cluster. Also Patroni will set them on every object (Endpoint or ConfigMap) it creates.
 -  **scope\_label**: (optional) name of the label containing cluster name. Default value is `cluster-name`.
--  **role\_label**: (optional) name of the label containing role (master or replica). Patroni will set this label on the pod it runs in. Default value is ``role``.
+-  **role\_label**: (optional) name of the label containing role (master or replica or other custom value). Patroni will set this label on the pod it runs in. Default value is ``role``.
+-  **leader\_label\_value**: (optional) value of the pod label when Postgres role is ``master``. Default value is ``master``.
+-  **follower\_label\_value**: (optional) value of the pod label when Postgres role is ``replica``. Default value is ``replica``.
+-  **tmp_\role\_label**: (optional) name of the label containing role (master or replica). Patroni will set this label on the pod it runs in. Default value is role_label thus no extra label will set.
```

Comment:
The default should be nothing, and when no value is set it shouldn't be even used here: https://github.com/zalando/patroni/pull/2659/files#diff-0b47c783038695efd3a30f9797eb9699abc996d28c5ec9f040b81e63e995b068R1290